renderdoc/driver/d3d12/d3d12_shader_feedback.cpp

```diff
@@ -892,23 +1222,32 @@ void D3D12Replay::FetchShaderFeedback(uint32_t eventId)
     return;
   }
 
-  bytebuf editedBlob[5];
+  bytebuf editedBlob[6];
```

Comment:
The idea is to have one index for each stage to simplify indexing farther in the function when adding direct access usages.

You are right, it would be better to use the same value as other parts of the codebase, I think it would be better to replace the hardcoded values and use the enum ShaderStage instead.